File: WHC_QRCodeScanKit(OC)/WHC_QRCodeScanKit(OC)/QRCodeWebVC.m
//
//  QRCodeWebVC.m
//  WHC_QRCodeScanKit(OC)
//
//  Created by 吴海超 on 15/12/29.
//  Copyright © 2015年 吴海超. All rights reserved.
//

#import "QRCodeWebVC.h"

@interface QRCodeWebVC ()
@property (nonatomic ,strong)IBOutlet UIWebView * webView;
@end

@implementation QRCodeWebVC

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    NSURL * url = [NSURL URLWithString:_value];
    NSURLRequest * urlRequest = [NSURLRequest requestWithURL:url];
    [_webView loadRequest:urlRequest];
}

@end

File: WHC_QRCodeScanKit/QRCodeWebVC.swift
//
//  QRCodeWebVC.swift
//  WHC_QRCodeScanKit
//
//  Created by 吴海超 on 15/12/29.
//  Copyright © 2015年 吴海超. All rights reserved.
//

import UIKit

class QRCodeWebVC: UIViewController {

    @IBOutlet var webView: UIWebView!
    var value: String!
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder);
    }
    
    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil);
    }
    
    convenience init(nibName: String){
        self.init(nibName: nibName,bundle: NSBundle.mainBundle());
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    override func viewWillAppear(animated: Bool) {
        super.viewWillAppear(animated)
        let url = NSURL(string: value)
        let urlRequest = NSURLRequest(URL: url!)
        webView.loadRequest(urlRequest)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
